irmaseal-pkg/src/server.rs

```diff
@@ -35,20 +52,33 @@ pub async fn exec(server_opts: ServerOpts) {
         sk: cgwkv_read_sk(secret).unwrap(),
     };
 
+    env_logger::init_from_env(env_logger::Env::new().default_filter_or("info"));
+
     HttpServer::new(move || {
         App::new()
+            .wrap(
+                Logger::new(
+                    "request=%{PATH}xi, status=%s, client=%{CLIENT_ID}xi, response_time=%D ms",
+                )
+                .custom_request_replace("CLIENT_ID", client_version)
+                .custom_request_replace("PATH", |req| {
+                    req.match_pattern().unwrap_or("-".to_string())
```

Comment:
I think it's better to not log paths that aren't matched. They might contain tokens.

Reply:
According to the documentation, "-" is convention.